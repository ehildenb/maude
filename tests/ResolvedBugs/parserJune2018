#!/bin/sh

MAUDE_LIB=$srcdir/../../src/Main
export MAUDE_LIB

../../src/Main/maude \
  < $srcdir/parserJune2018.maude -no-banner -no-advise \
  > parserJune2018.out 2>&1

diff $srcdir/parserJune2018.expected parserJune2018.out > /dev/null 2>&1
