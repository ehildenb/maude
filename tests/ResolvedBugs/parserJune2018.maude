set show advisories off .

*** Make sure syntax error is reported in the correct place (after b).

fmod FOO is
  sort Foo .
  op a_ : Foo -> Foo [prec 1 gather(e)] .
  op b_ : Foo -> Foo [prec 1] .
  op c : -> Foo .
endfm

parse a b c .
