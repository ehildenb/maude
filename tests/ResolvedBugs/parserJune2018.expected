Warning: <standard input>, line 12: didn't expect token b:
a b <---*HERE*
Warning: <standard input>, line 12: no parse for term.
Bye.
